src/Files.App/Views/Properties/MainPropertiesPage.xaml

```diff
@@ -73,8 +120,8 @@
 
 								<!--  Icon  -->
 								<Viewbox
-									Width="16"
-									Height="16"
+									Width="18"
+									Height="18"
```

Comment:
Can you share a screenshot of the before and after?

Reply:
Default navviewitem glyph is 18. but those icons were made for 16px, and so are blurry(Would you mind if I ask you to redesign to fit 18px?).

![image](https://user-images.githubusercontent.com/62196528/232266235-4e30bd91-96a2-43ab-978a-d9dcd9d2f1c1.png)

Reply:
We don't be able to redo them in time for this PR so I think we should revert the change. Would it help if we reduced the heigh of the navigation button?